.circleci/config.yml

```diff
@@ -70,20 +65,23 @@ commands:
 
 jobs:
   checkout_code:
-    environment:
-      RUBY_VERSION: << parameters.ruby_version >>
     parameters:
       ruby_version:
         type: string
+      elasticsearch_version:
+        type: string
 
-    executor: test_executor
+    executor: 
+      name: test_executor
+      ruby_version: << parameters.ruby_version >>
+      elasticsearch_version: << parameters.elasticsearch_version >>
 
     steps:
       - checkout
 
       - run:
           name: Wait for Elasticsearch 6.8
```

Comment:
@MothOnMars Minor nit that can be ignored if you wish, but seeing this "Wait for Elasticsearch 6.8" step logged out in a ES7 run is a bit confusing.  Do we need this at all?  Does it need to be conditional on the elasticsearch_version parameter?  Everything runs/passes/fails as anticipated, of course, so more of a "this just looks weird" comment.

Reply:
Good catch. We do need that step, but I'll rename it to a generic `Wait for Elasticsearch`.